src/main/java/frc5124/robot2020/commands/auto/runpos/RunDistance.java

```diff
@@ -9,6 +9,18 @@
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
 import frc5124.robot2020.subsystems.DriveTrain;
+
```

Comment:
has RunDistance been tested yet? is it supposed to be part of this turret and shooter PR?